### PR TITLE
[#1795] Fixed MODFLOW ModelInput resourcemetadata.xml issue

### PR DESCRIPTION
**Pull request by mohamedmorsy**

### CODE REVIEW

**Comment by mohamedmorsy** [comment]
@pkdash could you please review?

File: hs_modflow_modelinstance/models.py [hunk 1]
```diff
@@ -818,8 +818,9 @@ def get_xml(self, pretty_print=True):
                                              modelCalibrationFields)
 
         if self.model_inputs:
-            modelInputFields = ['inputType', 'inputSourceName', 'inputSourceURL']
-            self.add_metadata_element_to_xml(container, self.model_inputs.first(), modelInputFields)
+            for model_input in self.model_inputs:
+                modelInputFields = ['inputType', 'inputSourceName', 'inputSourceURL']
```
Comment (pkdash) [inline hs_modflow_modelinstance/models.py]: This line should go before the ```for``` loop.

Comment (mohamedmorsy) [inline hs_modflow_modelinstance/models.py]: @pkdash done

**Comment by pkdash** [comment]
+1

**Pull request merged**